### PR TITLE
Feature/profile

### PR DESCRIPTION
**Pull request by HarikrishnaSPillai**
Changes to Profile. 
Added projects that are done. 
Updated education

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,8 +1,72 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Harikrishna S Pillai . Jarvis Consulting
 
+A Software engineering enthusiast who is passionate in developing applications. Graduated in 2020 from Centennial College with focus on embedded systems where I developed several programs written in assembly language which communicated with microcontrollers. Working at Jarvis helped me realize my passion for programming and developing real world applications. Looking to leverage my acumen by working on challenging products and services.
```
Comment (jarviscanada) [inline README.md]: https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

File: README.md [hunk 2]
```diff
@@ -1,8 +1,72 @@
...
+**Competent:** Python, Tableau, MS Excel, Dart, Objective C
+
+**Familiar:** C++, JavaScript, CSS, HTML
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai)
+
+
+**Linux Cluster Monitoring Agent** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/linux_sql)]: The monitoring agent does the job of actively keeping an eye on the usage statistics which helps in effective resource management.This is possible with help of several different technologies working in tandem. The monitoring agent works on each node inserting values every minute from the node into postgres SQL database which mounted on a container using Docker.
```
Comment (jarviscanada) [inline README.md]: The monitoring agent does the job of actively keeping an eye on the usage statistics which helps in effective resource management. This is possible with help of several different technologies working in tandem. The monitoring agent works on each node inserting values every minute from the node into postgres SQL database which mounted on a container using Docker.

lease include the following keywords: CentOS, Bash/Shell, docker, RDBMS/SQL/Postgresql, crontab, data analytics/SQL


Developed XYZ for purposes using A, B, C.

File: README.md [hunk 3]
```diff
@@ -1,8 +1,72 @@
...
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/core_java)]:
+
+- **Grep App** : The grep Java application is the java implementation of the much popular 'grep' UNIX command. The program scans through files that are in directory and looks for a Regex Pattern (Regular Expression). Once matched copies all those matching lines and saves it to text file that can be specified when calling the program.
```
Comment (jarviscanada) [inline README.md]: Re-implemented the Linux grep command using Core Java....

      # Keywords: Java 8, Lambda, Stream API, IntelliJ

File: README.md [hunk 4]
```diff
@@ -1,8 +1,72 @@
...
+**Springboot App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/springboot)]: In Progress
+
+**Python Data Analytics** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/python_data_anlytics)]: In Progress
+
+**Hadoop** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/hadoop)]: In Progress
+
+**Spark** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/spark)]: In Progress
+
+**Cloud/DevOps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/cloud_devops)]: In Progress
+--->
```
Comment (jarviscanada) [inline README.md]: remove from YAML

File: README.md [hunk 5]
```diff
@@ -1,8 +1,72 @@
...
+
+## Highlighted Projects
+**Employee Trends of Salary, Gender Equality** [[GitHub](https://github.com/HarikrishnaSPillai/SQL-employees)]: The project involves efficient use of DML to extract insights from thousands of records to showcase how metrics like age, gender equality and salaries have progressed through years.The data which was extracted was visualized using Tableau.
+
+**Real Time Parking Spot Detection** [[GitHub](https://github.com/HarikrishnaSPillai/)]: Designed and Developed a system where Raspberry Pi works with TensorFlow to calculate the number of parking spots available, the data thus obtained is sent to users with Firebase and a Flutter app(Google Auth and Map API) on the client side
```
Comment (jarviscanada) [inline README.md]: fix github link

File: README.md [hunk 6]
```diff
@@ -1,8 +1,72 @@
...
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2021-present)**:
+- Designed and Developed BashScripts to run with Postgres database Image and Docker Engine.
+- Implemented apps in Java and created corresponding Docker images for portability
+- Proficient in using git and GitHub, praticed gitflow and efficient usage of branches to provide non-hinderance in master branches while delivering new products
```
Comment (jarviscanada) [inline README.md]: responsibilities, agile, soft skills

**Review by jarviscanada — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (HarikrishnaSPillai) [reply README.md]: Updated the whole paragraph.

File: README.md (hunk 2 above)
Comment (HarikrishnaSPillai) [reply README.md]: Included all the mentioned keywords

File: README.md (hunk 3 above)
Comment (HarikrishnaSPillai) [reply README.md]: Keywords included Java 8, Lambda and Stream, IntelliJ

File: README.md (hunk 4 above)
Comment (HarikrishnaSPillai) [reply README.md]: Removed from YAML

File: README.md (hunk 5 above)
Comment (HarikrishnaSPillai) [reply README.md]: The project was not issued on GitHub, Removed git-hub link.

File: README.md (hunk 6 above)
Comment (HarikrishnaSPillai) [reply README.md]: Included scrum and other keywords

File: README.md [hunk 7]
```diff
@@ -1,8 +1,47 @@
-# Jarvis Data Engineering Training
-1. [Linux cluster monitoring agent (Linux and SQL)](./linux_sql) In-progress
-2. [Core Java Apps](./core_java) In-progress
-3. [Springboot Trading REST API](./springboot) In-progress
-4. [Hadoop/Hive](./hadoop) In-progress
-5. [Spark/Scala](./spark) In-progress
-6. [Cloud/DevOps](./cloud_devops) In-progress
+# Harikrishna S Pillai . Jarvis Consulting
 
+A software enthusiast who is passionate about developing applications. During my academic years, I developed several projects using technologies like Flutter, Firebase, and Google APIs. Thereafter, pursuing a career in software became my prime focus. Recently, I started working at Jarvis as a Data Engineer. At Jarvis, our team used the Agile methodology which helped me align with industry-standard SDLC. Moreover, I was able to put my time-management, problem-solving and organizational skills to work by developing applications. Implementation of these applications helped me become proficient in technologies like Version Control, Java, Bash Scripting, RDBMS(SQL), Maven, Docker, JDBC, CLI. I aspire to deliver world-class applications along with pursuing my passion to learn new technologies.
```
Comment (jarviscanada) [inline README.md]: A software enthusiast who is passionate about developing applications. During my academic years, I developed several projects using technologies like Flutter, Firebase, and Google APIs. Thereafter, pursuing a career in software became my prime focus. Recently, I started working at Jarvis as a Data Engineer. At Jarvis, our team used the Agile methodology which helped me align with industry-standard SDLC. Moreover, I was able to put my time-management, problem-solving and organizational skills to work by developing applications. Implementation of these applications helped me become proficient in technologies like Java, Linux/Shell, GIT, RDBMS(SQL), Maven, Docker, and JDBC. I aspire to deliver world-class applications along with pursuing my passion for learning new technologies.

File: README.md [hunk 8]
```diff
@@ -1,8 +1,47 @@
...
+
+
+## Highlighted Projects
+**Real-Time Parking Detection System**: Designed and Developed a Parking Detection system in a Raspberry Pi and Intel Neural Compute Stick. The system used Yolo v3 Real-time Object detection algorithm to process images supplied by Pi cam. The number of parking spots was conveyed from Raspberry Pi to Firebase using Python and Web API. The data received in real-time was broadcast to the client-side web application. The web app was developed using Flutter and Dart programming language. Additionally, the web app used google authentication API and Map API for authentication and GUI purposes.
+
+**Portfolio Website**: Developed a Responsive website using tech HTML/CSS/JS as front end stack. The back-end was configured using AWS services like S3 and route 53. www.analysthari.com
```
Comment (jarviscanada) [inline README.md]: Developed a Responsive website using tech HTML/CSS/JS as front end stack. The website is hosted using AWS services like S3 and route 53. www.analysthari.com

Bootstrap, Angular/React??

File: README.md [hunk 9]
```diff
@@ -1,8 +1,47 @@
...
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Practised Scrum framework for team collaboration and SDLC. Developed several application using technologies such as Docker, Bash scripting, Java, RDBMS(SQL), Project management using Maven and version control using Git/GitHub.
```
Comment (jarviscanada) [inline README.md]: https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

File: README.md [hunk 10]
```diff
@@ -1,8 +1,47 @@
...
+
+**Product Analyst, BSC Global (2020-2021)**: Delivered quality reports on a biweekly basis using MS excel and Power BI to stakeholders to showcase the improvements made by the department. Prepared an informative dashboard for the logistics department which increased productivity by approximately 10 percent. Improvised the current SOP which increased the productivity by 25 percent with almost similar or decreased human effort.Test and Diagnose products with proper documentation to help organize inventory
```
Comment (jarviscanada) [inline README.md]: Delivered quality reports on a biweekly basis using MS Excel and Power BI to stakeholders to showcase the improvements made by the department. Prepared an informative dashboard for the logistics department which increased productivity by approximately 10 percent. Improvised the current SOP which increased the productivity by 25 percent with almost similar or decreased human effort. Test and Diagnose products with proper documentation to help organize inventory

**Review by jarviscanada — APPROVED** [review]

File: README.md (hunk 10 above)
Comment (HarikrishnaSPillai) [reply README.md]: Changed to MS Excel

File: README.md (hunk 7 above)
Comment (HarikrishnaSPillai) [reply README.md]: Reordered the technologies as suggested

File: README.md (hunk 9 above)
Comment (HarikrishnaSPillai) [reply README.md]: Increased the word count in Experiences

File: README.md (hunk 8 above)
Comment (HarikrishnaSPillai) [reply README.md]: Included more tech stack which was used in the project

**Pull request merged**